testing/camunda-process-test-java/src/main/java/io/camunda/process/test/impl/extension/CamundaProcessTestContextImpl.java

```diff
@@ -87,6 +87,7 @@ public CamundaClient createClient() {
   @Override
   public CamundaClient createClient(final Consumer<CamundaClientBuilder> modifier) {
     final CamundaClientBuilder builder = camundaClientBuilderFactory.get();
+    builder.preferRestOverGrpc(false);
```

Comment:
❓ Is this necessary? What is CPT doing that requires gRPC? 🤔

Reply:
locally it works also with Rest 🤔 but I had some CI failures - maybe they were flakes. Still, I would leave it on grpc for now as before it was also using grpc. I think we should invest some more time in the feature to have all/most tests also running with Rest (maybe via parameterized tests)